tests/test_toa_indexing.py

```diff
@@ -27,9 +27,6 @@ def test_get_TOAs(high_precision):
     toas = get_TOAs(StringIO(tim), ephem="de421")
     m = toas.get_mjds(high_precision=high_precision)
     assert isinstance(m, np.ndarray)
-    assert not np.all(
-        np.diff(m) > 0
-    ), "returned values should be grouped by observatory"
```

Comment:
This should become a test that indeed all returned values are increasing (assuming that's what's in the input par).